src/features/users/user-controller.ts

```diff
@@ -334,7 +334,7 @@ class UserController {
             await emailHelper.sendEmail({
                 content: `Hello,
 
-                Please verify your account by clicking this url: ${baseUrl}/verify/${newUser.verifyToken}
+                Please verify your account by clicking this url: ${baseUrl}/backend-api/users/verify?verifyToken=${newUser.verifyToken}
```

Comment:
This is the incorrect url, this will point to the backend but we need the front end to make the request so that they can render the page and present an error message if required

Reply:
also, I assumed an incorrect `${baseUrl}` in my previous comment...